contents/forward_euler_method/code/java/Euler.java

```diff
@@ -0,0 +1,38 @@
+public class Euler {
+    private static double[] solveEuler(double timestep, int n) {
+        double[] eulerResult = new double[n];
+
+        //Setting the initial condition
+        eulerResult[1] = 1;
+        for(int i = 2; i < eulerResult.length; i++) {
+            eulerResult[i] = eulerResult[i-1] - (3 * eulerResult[i-1] * timestep);
```

Comment:
Two things in thise line:

1. The parentheses around `(3 * eulerResult[i-1] * timestep)` aren't needed, although I know why you added them. They make the expression a bit more readable and I think I agree, so I guess we'll keep it.

2. You didn't space out `[i-1]` and again, I understand why. But in this case, I think I like `[i - 1]` more. What do you think?

```suggestion
            eulerResult[i] = eulerResult[i - 1] - (3 * eulerResult[i - 1] * timestep);
```